include/tvm/relax/analysis.h

```diff
@@ -404,6 +404,18 @@ TVM_DLL Map<Var, Array<Var>> DataflowBlockUseDef(const DataflowBlock& dfb);
  */
 std::pair<Map<Var, Array<Var>>, Array<Var>> FunctionUseDef(const Function& fn);
 
+/*!
+ * \brief Perform a liveness analysis on the function, indicating which variables
+ * are live at which location in the function.
+ *
+ * \param fn The function to be analyzed.
```

Comment:
It depends on whether we plan to reuse the dataflow analysis framework (I was hoping to but it doesn't sound like there's an immediate candidate for another analysis we would implement with it). If so, the CFG would be needed for many things and we probably wouldn't want to make all of them methods of the class.

Reply:
Good point.  I usually lean toward member functions, because they are much easier for a developer to discover, and avoid requiring long names to provide enough context to a reader.  It would be nice if there were something similar to the extension methods of C# or Rust, to allow the improved readability without making monolithic classes.